lamb/README.md

```diff
@@ -0,0 +1,99 @@
+![Lamb](https://github.com/scalar-labs/kelpie-test/workflows/Lamb/badge.svg)
+
+# Kelpie Lamb
+Kelpie Lamb is a Kelpie extension for benchmarking of Scalar DL contracts.
+
+## Build
+
+```console
+$ ./gradlew shadowJar
+```
+
+## How to run a test
+Before benchmarking, you first have to set up your Scalar DL environment.
```

Comment:
This sentence looks like a summary of the following items, but it's not. 
I feel this section should be moved to a prerequisite section.

Reply:
I added the Prerequisite section